.env.example

```diff
@@ -2,3 +2,4 @@ PRIVATE_KEY="ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
 SNOWTRACE_API_KEY="<YOUR SNOWTRACE KEY>"
 RPC_MAINNET_URL="https://api.avax.network/ext/bc/C/rpc"
 RPC_TESTNET_URL="https://api.avax-test.network/ext/bc/C/rpc"
+FOUNDRY_OUT=abi
```

Comment:
Uhhh there's so much crap in the foundry output folder....

I think a better solution would be to write a script that copies the abis (and only the relevant ones) to the abi folder. It should probably also only copy the insides of the "abi" key to ignore all the other crap that's changing constantly.

This could maybe be run as a part of the build script.

Alternatively, configure foundry so that the abis don't change every time and gitignore all the irrelevant stuff from `abi/`.

Reply:
resolved - .gitignore abi